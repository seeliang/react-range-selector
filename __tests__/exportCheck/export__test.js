import RangeSelector from '../../lib/index.js';
import renderer from 'react-test-renderer';
import React from 'react';

it('renders range', () => {
  const tree = renderer.create(
    <RangeSelector componentName={'range-selector'}
      key={3}
      name={'car'}
      range={[3,5]}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
