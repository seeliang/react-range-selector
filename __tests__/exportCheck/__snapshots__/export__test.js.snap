// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders range 1`] = `
<div
  className="range-selector__section"
>
  <b
    className="range-selector__section-title"
  >
    car
  </b>
  <div
    className="range-selector__item"
  >
    <input
      checked={false}
      className="range-selector__item-checkbox"
      id="range-selector_car-0"
      onChange={[Function]}
      type="checkbox"
      value="range-selector_car-0"
    />
    <label
      className="range-selector__item-label "
      htmlFor="range-selector_car-0"
    >
      3
    </label>
  </div>
  <div
    className="range-selector__item"
  >
    <input
      checked={false}
      className="range-selector__item-checkbox"
      id="range-selector_car-1"
      onChange={[Function]}
      type="checkbox"
      value="range-selector_car-1"
    />
    <label
      className="range-selector__item-label "
      htmlFor="range-selector_car-1"
    >
      4
    </label>
  </div>
  <div
    className="range-selector__item"
  >
    <input
      checked={false}
      className="range-selector__item-checkbox"
      id="range-selector_car-2"
      onChange={[Function]}
      type="checkbox"
      value="range-selector_car-2"
    />
    <label
      className="range-selector__item-label "
      htmlFor="range-selector_car-2"
    >
      5
    </label>
  </div>
</div>
`;
